chapters/VK_EXT_acquire_wl_display/acquire_wl_display.txt

```diff
@@ -0,0 +1,66 @@
+[open,refpage='vkAcquireWaylandDisplayEXT',desc='Acquire access to a VkDisplayKHR using Wayland',type='protos']
+--
+
+To acquire permission to directly access a display in Vulkan from a Wayland
+compositor, call:
+
+include::{generated}/api/protos/vkAcquireWaylandDisplayEXT.txt[]
+
+  * pname:physicalDevice The physical device the display is on.
+  * pname:display A wl_display connected to the Wayland compositor that
```

Comment:
typically this will be `sname:wl_display` for a structure-ish type, but it's probably not critical. (Just don't use the `*link:` markup macros, those are for internally-defined API only.)

Reply:
Nope, `sname` is for Vulkan types.

See this https://github.com/KhronosGroup/Vulkan-Docs/blob/6912eb5d88bf61d6a6e8be8493d2ed8e5c887f4e/chapters/VK_KHR_wayland_surface/platformCreateSurface_wayland.txt#L45

Reply:
Ah, thanks for the clarification. The usage isn't entirely consistent, and differs slightly from the OpenXR usage (my "native" WG).

Reply:
Seems like we need a specification for making specifications :p

Reply:
Well, there's the style guide, but the curse of such documents is that it's typically always slightly out of date. This is especially true for OpenXR which started with a Vulkan fork and isn't quite updated yet (and thus isn't fully public yet)